### ISSUE TITLE
Add a configurable container class in the default config

### ISSUE THREAD

**Issue by perruche** [studiometa/tailwind-config#62]
For exemple:
```
.container {
  width: 100%;
  max-width: 80rem;
  margin-right: auto;
  margin-left: auto;
  padding-right: 1.5rem;
  padding-left: 1.5rem;

  @screen s {
    padding-right: 2rem;
    padding-left: 2rem;
  }
}
```

**Comment by antoine4livre** [comment]
@perruche Does the PR #94 responses to your needs ?

**Comment by antoine4livre** [comment]
Available in v2.1.0 🎆
